src/main/java/nextstep/ladder/domain/ladder/Rung.java

```diff
@@ -0,0 +1,40 @@
+package nextstep.ladder.domain.ladder;
+
+import java.util.Optional;
+
+/**
+ * 사다리 가로대(발판)
+ */
+public enum Rung {
+    EXIST(true),
+    EMPTY(false),
+    ;
+
+    private final boolean exist;
+
+    Rung(boolean exist) {
+        this.exist = exist;
+    }
+
+    public static Rung findByBoolean(boolean value) {
```

Comment:
```suggestion
    public static Rung from(boolean value) {
```
매개변수 타입으로 유추할 수 있기 때문에 메소드에 자료형을 명시할 필요가 있을까? 하는 생각이 듭니다.
관익님은 어떻게 생각하시나요?

Reply:
사실 처음에 from으로 구현했었는데 뭔가 내적 갈등을 하다가 바꿨던 것 같네요..